tests/integration/container_test.py

```diff
@@ -382,6 +382,21 @@ def test_create_with_environment_variable_no_value(self):
             sorted(['Foo', 'Other=one', 'Blank='])
         )
 
+    def test_create_with_tmpfs(self):
+        tmpfs = {
+            '/tmp1': 'size=3M'
+        }
+
+        container = self.client.create_container(
+            BUSYBOX,
+            ['echo'],
+            host_config=self.client.create_host_config(
+                tmpfs=tmpfs))
+
+        self.tmp_containers.append(container['Id'])
+        config = self.client.inspect_container(container)
+        assert config['HostConfig']['Tmpfs'] == tmpfs
```

Comment:
This is failing with a KeyError for some reason, I'm not sure why, it seems to be correct to me.